src/main/scala/de/htwg/se/sudoku/Sudoku.scala

```diff
@@ -11,7 +13,11 @@ object Sudoku {
   val injector: Injector = Guice.createInjector(new SudokuModule)
   val controller: ControllerInterface = injector.getInstance(classOf[ControllerInterface])
   val tui = new Tui(controller)
-  val gui = new SwingGui(controller)
+
+  if (!GraphicsEnvironment.isHeadless) {
```

Comment:
Had to introduce this check because there isn't any UI available using docker and Swing doesn't handle this well, it throws only a HeadlessException